pkg/liqonet/overlay_network.go

```diff
@@ -103,7 +103,7 @@ func Enable_rp_filter() error {
 	for index := range ifaces_list {
 		if ifaces_list[index].Type() == "vxlan" {
 			// Enable loose mode reverse path filtering on the vxlan interface.
-			err = ioutil.WriteFile("/proc/sys/net/ipv4/conf/"+ifaces_list[index].Attrs().Name+"/rp_filter", []byte("2"), 0644)
+			err = ioutil.WriteFile("/proc/sys/net/ipv4/conf/"+ifaces_list[index].Attrs().Name+"/rp_filter", []byte("2"), 0600)
```

Comment:
@alacuku gosec suggested this permission change. Does it sound ok to you?

Reply:
well the default permission of that file are `0644 (-rw-r--r--)` with `user root` and `group root`. what gosec is suggesting is to remove the read permissions of all the users belonging to the` root group` and also to the `rest of the world`. Being it a system file i would maintain the default permission.

Reply:
0644 seems to be in fact [the default permission for the root user](https://www.cyberciti.biz/tips/understanding-linux-unix-umask-value-usage.html). Nevertheless also *00 appears to be a common configuration and maybe restricting the privileges a little more as gosec suggested could be a good idea. Do you think it could create troubles?

Reply:
We discussed with @alacuku, and we agreed that this should not break. I'll test it on a cluster before merging.